### DISCUSSION TITLE
Delete Sensors from Sensor list

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#1143] [Q&A] [answered]
![Sensors to delete](https://github.com/mytourbook/mytourbook/assets/106929478/860e755a-9673-4e9f-b9ca-82a9fdf6d835)
The above four sensors where an artifact of loading two friends' tours into a "Test" part of my database.  After I removed those tours the sensors persisted with dates 1Jan1970.  I attempted to remove those sensors by deleting the lucene-index folder from the derby-database as suggested for prior inconsistencies.  However that made no difference.

1. How do I delete (censor - haha)  sensors that were removed from all tours?
2. Should I post this as an enhancement request?

**Comment by wolfgang-ch** [comment]
> I attempted to remove those sensors by deleting the lucene-index folder from the derby-database as suggested for prior inconsistencies.

A device sensor is not stored in the lucene-index, it's stored in the SQL database similar like a tour marker

>    How do I delete (censor - haha) sensors that were removed from all tours?

This is currently not supportet

>    Should I post this as an enhancement request?

Yes

**Comment by wolfgang-ch** [comment, marked as answer]
This is implemented in MT 23.8

**Reply by stvzldn** [reply]
Installed 23.8 and was able to delete the unwanted sensors and edit other types.
